docs/how-to-use-gvfs.md

```diff
@@ -0,0 +1,145 @@
+---
+title: How to use Gravitino Virtual File System
+slug: /how-to-use-gvfs
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Introduction
+
```

Comment:
The Gravitino Virtual File System (Gvfs) is a distributed file system mounted into a Gravitino metalake. Gvfs is an abstraction on top of scalable object storage that maps Unix-like filesystem calls to native cloud storage API calls.